bundle/05_spleen_segmentation_lightning.ipynb

```diff
@@ -0,0 +1,1126 @@
+      "2023-10-17 16:48:01,076 - INFO - > config_file: ['./SpleenSegLightning/configs/common.yaml',\n",
+      " './SpleenSegLightning/configs/evaluate.yaml']\n",
+      "2023-10-17 16:48:01,076 - INFO - > meta_file: './SpleenSegLightning/configs/metadata.json'\n",
+      "2023-10-17 16:48:01,076 - INFO - > run_id: 'evaluate'\n",
+      "2023-10-17 16:48:01,076 - INFO - > bundle_dir: './SpleenSegLightning'\n",
+      "2023-10-17 16:48:01,076 - INFO - > data_dir: './Task09_Spleen'\n",
+      "2023-10-17 16:48:01,076 - INFO - > ckpt_file: './epoch=599-step=9600.ckpt'\n",
+      "2023-10-17 16:48:01,076 - INFO - ---\n",
+      "\n",
+      "\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "monai.bundle.workflows ConfigWorkflow.__init__:workflow_type: Current default value of argument `workflow_type=None` has been deprecated since version 1.2. It will be changed to `workflow_type=train` in version 1.4.\n",
+      "Default logging file in SpleenSegLightning/configs/logging.conf does not exist, skipping logging.\n",
+      "Loading dataset: 100%|██████████| 9/9 [00:10<00:00,  1.12s/it]\n",
+      "GPU available: True (cuda), used: True\n",
+      "TPU available: False, using: 0 TPU cores\n",
+      "IPU available: False, using: 0 IPUs\n",
+      "HPU available: False, using: 0 HPUs\n",
+      "Restoring states from the checkpoint path at ./epoch=599-step=9600.ckpt\n",
+      "LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0,1]\n",
+      "Loaded model weights from the checkpoint at ./epoch=599-step=9600.ckpt\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Validation DataLoader 0: 100%|██████████| 9/9 [00:17<00:00,  1.90s/it]current epoch: 0 current mean dice: 0.9123\n",
+      "best mean dice: 0.9123 at epoch: 0\n",
+      "Validation DataLoader 0: 100%|██████████| 9/9 [00:17<00:00,  1.90s/it]\n"
+     ]
+    }
+   ],
+   "source": [
+    "%%bash\n",
+    "\n",
+    "DATA_DIR=\"./Task09_Spleen\"\n",
+    "BUNDLE=\"./SpleenSegLightning\"\n",
+    "export PYTHONPATH=\"$BUNDLE\"\n",
+    "\n",
+    "python -m monai.bundle run evaluate \\\n",
+    "    --bundle_dir \"$BUNDLE\" \\\n",
+    "    --data_dir \"$DATA_DIR\" \\\n",
+    "    --meta_file \"$BUNDLE/configs/metadata.json\" \\\n",
+    "    --config_file \"['$BUNDLE/configs/common.yaml','$BUNDLE/configs/evaluate.yaml']\" \\\n",
+    "    --ckpt_file \"./epoch=599-step=9600.ckpt\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6fd62905-4ea8-4f08-bcea-823074fc4ce4",
+   "metadata": {},
+   "source": [
+    "## Summary and Next\n",
+    "\n",
+    "This tutorial has covered:\n",
+    "* Creating full training and evaluation scripts in bundles using MONAI and Pytorch Lightning\n",
+    "* Training a network then evaluating its performance with scripts.\n",
+    "\n",
+    "The next tutorial will discuss creating bundles to combine Pytorch Lightning CLI config files with MONAI bundles."
```

Comment:
Let's remove this line since the next tutorial isn't planned yet.

Reply:
Good point! Thank you! I removed that in the new version notebook.